### PR TITLE
documentation for opcodes in English

### PR DESCRIPTION
**Pull request by igormcoelho**
@erikzhang I documented the opcodes (in English), and many of them were lacking any type of documentation. If you think that's correct, please accept this PR.

### CODE REVIEW

**Comment by shargon** [comment]
Please use ```/// <sumary>```

**Comment by vncoelho** [comment]
Nice comments, broda. 

@shargon, you mean like this:

```
/// <summary>
///  An array is removed from top of the main stack. Its size is put on top of the main stack.
/// </summary>
ARRAYSIZE = 0xC0,     
```

**Comment by shargon** [comment]
Yes, if you do this Visual studio should show this comments as tooltip, is useful

**Comment by vncoelho** [comment]
Thank you, Shargon.
We are on MonoDevelop, but should be the same.

Soon we gonna adjust it.

**Comment by shargon** [comment]
@vncohelo remember this :p

**Comment by vncoelho** [comment]
@shargon, soon we gonna finish it. Sorry for the delay, busy days, master.

**Comment by igormcoelho** [comment]
Ok @shargon, I think it's nice now with summary structure :)

File: src/neo-vm/OpCode.cs [hunk 1]
```diff
@@ -2,142 +2,460 @@ namespace Neo.VM
 {
     public enum OpCode : byte
     {
+      /// <summary>
+      ///
```
Comment (shargon) [inline src/neo-vm/OpCode.cs]: 👎  xD

Comment (igormcoelho) [reply src/neo-vm/OpCode.cs]: hahah that was my reference for copy and paste... forgot it on the final version :D

**Pull request merged**